doc/sphinx/changes.rst

```diff
@@ -8,6 +8,694 @@ Recent changes
+  (`#13490 <https://github.com/coq/coq/pull/13490>`_,
+  by Kazuhiko Sakaguchi).
+
+Commands and options
+^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  :cmd:`Hint Rewrite` now supports locality attributes (including :attr:`export`) like other :ref:`Hint <creating_hints>` commands
+  (`#13725 <https://github.com/coq/coq/pull/13725>`_,
+  fixes `#13724 <https://github.com/coq/coq/issues/13724>`_,
+  by Gaëtan Gilbert).
+- **Changed:**
+  In `Record`, alpha-rename the variable associated with the record to avoid
+  alpha-renaming parameters of projections
+  (`#13852 <https://github.com/coq/coq/pull/13852>`_,
+  fixes `#13727 <https://github.com/coq/coq/issues/13727>`_,
+  by Li-yao Xia).
+- **Changed:**
+  Improve the :cmd:`Coercion` command to reduce the number of ambiguous paths to
+  report. A pair of multiple inheritance paths that can be reduced to smaller
+  adjoining pairs will not be reported as ambiguous paths anymore.
+  (`#13909 <https://github.com/coq/coq/pull/13909>`_,
+  by Kazuhiko Sakaguchi).
+- **Changed:**
+  The printing order of :cmd:`Print Classes` and :cmd:`Print Graph`, due to the
+  changes for the internal tables of coercion classes and coercion paths.
+  (`#13912 <https://github.com/coq/coq/pull/13912>`_,
+  by Kazuhiko Sakaguchi).
+- **Removed:**
+  The Hide Obligations flag, deprecated in 8.12
+  (`#13758 <https://github.com/coq/coq/pull/13758>`_,
+  by Jim Fehrle).
+- **Removed:**
+  SearchHead command.  Use the `headconcl:` clause of :cmd:`Search` instead
+  (`#13763 <https://github.com/coq/coq/pull/13763>`_,
+  by Jim Fehrle).
+- **Removed:**
+  `Show Zify Spec`, `Add InjTyp` and 11 similar `Add *` commands.
+  For `Show Zify Spec`, use `Show Zify UnOpSpec` or `Show Zify BinOpSpec` instead.
+  For `Add *`, `Use Add Zify *` intead of `Add *`
+  (`#13764 <https://github.com/coq/coq/pull/13764>`_,
+  by Jim Fehrle).
+- **Deprecated:**
+  Like hints, typeclass instances added outside of sections
+  without an explicit locality now generate a deprecation warning. See
+  :ref:`Hint <creating_hints>`
+  (`#14208 <https://github.com/coq/coq/pull/14208>`_,
+  fixes `#13562 <https://github.com/coq/coq/issues/13562>`_,
+  by Pierre-Marie Pédrot).
+- **Deprecated:**
+  the :flag:`Regular Subst Tactic` flag
+  (`#14336 <https://github.com/coq/coq/pull/14336>`_,
+  by Pierre-Marie Pédrot).
+- **Added:**
+  :opt:`Debug` to control debug messages, functioning similarly to the warning system
+  (`#13202 <https://github.com/coq/coq/pull/13202>`_,
+  by Maxime Dénès and Gaëtan Gilbert).
+  The following flags have been converted (such that ``Set Flag`` becomes ``Set Debug "flag"``):
+  - ``Debug Unification`` to ``unification``
+  - ``Debug HO Unification`` to ``ho-unification``
+  - ``Debug Tactic Unification`` to ``tactic-unification``
+  - ``Congruence Verbose`` to ``congruence``
+  - ``Debug Cbv`` to ``cbv``
+  - ``Debug RAKAM`` to ``RAKAM``
+  - ``Debug Ssreflect`` to ``ssreflect``
```

Comment:
This doesn't render properly. Maybe it just needs a new line before the bullet-points.